CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.8.0] 2021-12-09
+
+### Changed
+
+- **BREAKING:** MongoDB driver and Tokio update
+  tokio 1.x
+  mongodb 2.0
```

Comment:
```suggestion
- **BREAKING:** Update `mongodb` to `2.x`
```

`tokio` is a transitive dependency, but we don't directly depend on it. I think it's enough to specify the `mongodb` update for the breaking change.

Reply:
Gotcha